bases/renku_data_services/data_api/main.py

```diff
@@ -200,4 +208,7 @@ async def logging_setup2(app: Sanic) -> None:
     loader = AppLoader(factory=create_app)
     app = loader.load()
     app.prepare(**args)
-    Sanic.serve(primary=app, app_loader=loader)
+    if os.name == "posix":
+        Sanic.serve(primary=app)
+    else:
+        Sanic.serve(primary=app, app_loader=loader)
```

Comment:
Why is this dependent on the `os` and not on the value of `single_process`? With this PR, it seems that the user has no choice but to use `--single-process` to run the server.

Reply:
Which is why we should set `spawn` or `fork` depending on the value of `--single-process`.

Reply:
Sanic recommends somewhere in the docs to change the start method as early as possible right after you import Sanic. So we will be breaking that recommendation. But I can try.